jinahub/indexers/merger/SimpleMerger/simple_merger.py

```diff
@@ -0,0 +1,34 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from collections import OrderedDict
+from typing import List, Tuple
+
+from jina import Executor, requests, DocumentArray
+
+
+class SimpleMerger(Executor):
```

Comment:
The name should be `MatchMerger`. Or we need to pass what to `merge` , `chunks` or `matches`

Reply:
Okay I will call it MatchMerger for now